x-pack/plugins/ml/public/url_generator.ts

```diff
@@ -24,11 +26,81 @@ declare module '../../../../src/plugins/share/public' {
 
 export const ML_APP_URL_GENERATOR = 'ML_APP_URL_GENERATOR';
 
+export const ML_PAGES = {
+  ANOMALY_DETECTION: 'jobs',
+  ANOMALY_EXPLORER: 'explorer',
+  TIME_SERIES_EXPLORER: 'timeseriesexplorer',
```

Comment:
I think this should be `SINGLE_METRIC_VIEWER` as that is the public facing name for this page.

Reply:
Updated here https://github.com/elastic/kibana/pull/75696/commits/d8c5716ee25a6f0d2ec4c79d7f6462a3e997546d